### PR TITLE
Create steel.ms

### PR DESCRIPTION
**Pull request by jaikev**

### CODE REVIEW

**Comment by Boofurd** [comment]
...Regen 10? That's literally faster than deadpool.

**Comment by jaikev** [comment]
He's from steel he can't even get hurt well he can a little but not very  much like one gun with a special thing on it

File: CommandHelper/LocalPackages/commands/buffs/steel.ms [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODU5NzIxODM4MTEsInByb2ZpbGVJZCI6IjU1MGIwM2NkZDllOTRjODliNGVhNDRlYWQ0ODljZTBlIiwicHJvZmlsZU5hbWUiOiJBbmltZWxHaXJsIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS81M2Q4N2JjMDhlNGM4MDE2NDI3OGViNmRkOWQ0YzVkNGM2ZWRiZjJiYmQ4MWY0MzRmZDk4Y2I2MmRkNjE2MzE5In19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "My grand grand grand father was a hero in the past!",
+            'powers': array(
+                "/buff",
+                "/jump "0-3",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/steel.ms]: too many " on this line

File: CommandHelper/LocalPackages/commands/buffs/steel.ms [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                "/run "0-3",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/steel.ms]: too many " on this line

File: CommandHelper/LocalPackages/commands/buffs/steel.ms [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                "/haste "0-5",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/steel.ms]: too many " on this line

**Pull request merged**